cmd/ci-secret-bootstrap/main.go

```diff
@@ -480,30 +628,54 @@ func main() {
 			logrus.WithError(err).Fatal("Failed to logout.")
 		}
 	})
-	defer logrus.Exit(0)
```

Comment:
Is it moved somewhere?
I believe we still need it.

Reply:
Well, this change here is not related with my PR. However, this line can make the tool to exit 0 when the code panics and also the panic message is not even shown. This is why I removed it. Why we needed this from the first place?